README.md

```diff
@@ -6,14 +6,40 @@ on Content Defined Chunking (CDC), in particular
 to split up files into chunks.
 
 ## CDC RSync
-Tool to sync files to a remote machine, similar to the standard Linux
-[rsync](https://linux.die.net/man/1/rsync). It supports fast compression and
-uses a higher performing remote diffing approach based on CDC.
 
-## Asset Streaming
-Tool to stream assets from a Windows machine to a Linux device.
+CDC RSync is a ool to sync files to a remote machine, similar to the standard
```

Comment:
I think it is important to mention that it currently only supports a Windows -> Linux sync.

Reply:
Done.